### ISSUE TITLE
AutoPreviewImageUrl returns wrong multigeo tenant.

### ISSUE THREAD

**Issue by poneilcsgp** [microsoft-search/pnp-modern-search#3348]
__Remove the above section before submitting, and removal of the below template will result in the issue being closed.__

**Version used**
Ex: 4.9.3

**Describe the bug**
I'm pulling news from sites across different multigeo tenants.  The AutoPreviewImageUrl provides a link that refers to the wrong tenant when the item in the index is in a different tenant than the search result web part page.   This causes a broken link.   If I change the link to add the right tenant suffix back in the link works again.

**To Reproduce**
Detailed steps to reproduce the behavior:
1. Create a news article in one multigeo tenant (I used one other than our default).  Add an image to the page using the header image web part, etc.
2. Create a page on the default tenant, add a PnP Search Results webpart and select news stories (for example).    Choose the "debug" template and confirm that the image url in AutoPreviewImageUrl property is coming from the default tenant (rather than the other geo) and that url does not point to an item that exists.
3. add the geo suffix to the broken link above and confirm the image appears.

**Expected behavior**
The AutoPreviewImageUrl property should return the correct url to the image, even when being queried from another geo.

**Screenshots**
![Screenshot 2023-11-20 at 5 08 11 PM](https://github.com/microsoft-search/pnp-modern-search/assets/104784418/41394e57-03ea-40cb-af2e-2736b7fa5d80)
**Desktop (please complete the following information):**
 - Browser Chrome

**Additional context**
No additional context, but happy to provide answers to any follow on questions.  Thank you (as always) for your help!

**Comment by kasperbolarsen** [comment]
Can you check the values from the API using SP Editor or the SharePoint search query tool? 
If the API is returning bad data then we can't do anything,  except asking you to create an issue with the search team at MS

**Comment by kasperbolarsen** [comment]
I can see in https://github.com/microsoft-search/pnp-modern-search/issues/2893 that they switched to using PictureThumbnailURL  instead. Perhaps worth a try? 
I do not have access to a multi geo tenant so I can't recreate the scenario, sorry

**Comment by wobba** [comment]
Your workaround is not to use the auto slots, but rather use one of the managed properties manually, like `Path`.

**Comment by wobba** [comment]
Right....image and geo. There is no simple solution here except an extra component using e.g. Graph instead afaik. This was an internal MS issue as well which got solved by a middle-tier.

**Comment by poneilcsgp** [comment]
Makes sense, thanks for your help.  I'll see what I can do w/ search results to generate a request to graph.

**Comment by wobba** [comment]
Closing — AutoPreviewImageURL is returned by the Microsoft search API and the multigeo URL originates server-side, so there is nothing the web parts can change here. This should be raised with the Microsoft Search team. Please reopen if you can show the web part is altering the value. Thanks!
